files/usr/lib/systemd/system/soft-reboot-cleanup.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Cleanup system data after soft-reboot
+Before=network-pre.target
```

Comment:
According to https://systemd.io/NETWORK_ONLINE/, `network-pre.target` is not always pulled in, so this needs to be either

```
Before=network-pre.target network.target
```

or

```
Before=network-pre.target
Wants=network-pre.target
```